sound/soc/sof/intel/pci-lnl.c

```diff
@@ -36,7 +36,7 @@ static const struct sof_dev_desc lnl_desc = {
 		[SOF_IPC_TYPE_4] = "intel/sof-ipc4/lnl",
 	},
 	.default_tplg_path = {
-		[SOF_IPC_TYPE_4] = "intel/sof-ace-tplg",
+		[SOF_IPC_TYPE_4] = "intel/sof-ipc4-tplg",
```

Comment:
For IPC4 are there any specific modifications in topology? Or just the file location?

Will IPC4 work with topology1 files?

Thanks!

Reply:
IPC4 needs topology2 files. We want to keep these separate as we might have platforms (officially) supported by both IPC3 (topology1) and IPC4 (topology2) in upstream.

Reply:
@iuliana-prodan @ujfalusi There's no strict dependency, you can write IPC3 tplgs with tplg1 or tplg2, and in reverse, you can write IPC4 tplgs with tplg1 or tplg2. That said, available common macro definitions to define pipelines limit the options so that currently you'll have to use tplg2 for IPC4 and tplg1 for IPC3.

Reply:
@kv2019i, thanks for the correction!

Reply:
@ujfalusi Ack, the kernel will expect different entities in the tplg file for IPC3 and IPC4, so a single tplg file cannot provide both,.

Reply:
@kv2019i is there any example of tplg1 file working with IPC4?

Reply:
@iuliana-prodan Unfortunately no, all the Intel work for IPC4 was done with the tplg2 frontend.

Reply:
Thank you @kv2019i, @ujfalusi